paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -596,6 +596,41 @@
                             ]
                         }
                     }
+                },
+                "sidecar_resource_requirements": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "hacheck": {
+                            "type": "object",
+                            "properties": {
+                                "requests": {
```

Comment:
In addition to @stug's comment about the `type` field we probably need a `properties` field wrapping the cpu, memory and ephemeral-storage keys right.